pkgs/development/python-modules/hglib/default.nix

```diff
@@ -14,12 +14,21 @@ buildPythonPackage rec {
       src = ./hgpath.patch;
       hg = "${mercurial}/bin/hg";
     })
+    (fetchpatch {
+      url = "https://www.mercurial-scm.org/repo/python-hglib/raw-rev/12e6aaef0f6e";
+      sha256 = "159pmhy23gqcc6rkh5jrni8fba4xbhxwcc0jf02wqr7f82kv8a7x";
+    })
+    (fetchpatch {
+      url = "https://www.mercurial-scm.org/repo/python-hglib/raw-rev/1a318162f06f";
+      sha256 = "04lxfc15m3yw5kvp133xg8zv09l8kndi146xk3lnbbm07fgcnn1z";
+    })
```

Comment:
do you mind adding comments on why these are necessary, and when they can be removed?

Reply:
@jonringer I addressed your comments and force pushed an updated commit to the PR branch. Could you, please, take a look at my new changes and see if you find them more appropriate for inclusion to nixpkgs?